app/views/option_sets/_form.html.erb

```diff
@@ -17,7 +17,7 @@
     <%= f.field(:geographic, :type => :check_box, :read_only => cannot?(:change_geographic, @option_set)) %>
     <%= f.field(:allow_coordinates, :type => :check_box, :read_only => cannot?(:change_geographic, @option_set)) %>
     <%= f.field(:multi_level, :type => :check_box,
-      :read_only => cannot?(:update_core, @option_set) || @option_set.has_select_multiple_questions? || @option_set.huge?) %>
+      :read_only => cannot?(:update_core, @option_set) || @option_set.has_select_multiple_questions? || @option_set.huge? || params[:question_type] == "select_multiple") %>
```

Comment:
Great, Tom! 
The same concern crossed my mind when i saw all those checks on the template... Doing what you said will improve this a lot!

Will do these changes here... Thanks!